base_search_custom_field_filter/models/ir_ui_custom_field_filter.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Tecnativa - Carlos Dauden
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, exceptions, fields, models
+
+
+class IrUiCustomFilter(models.Model):
+    _name = "ir.ui.custom.field.filter"
+    _description = "Custom UI field filter"
+    _order = "sequence, model_id, id"
+
+    sequence = fields.Integer()
+    model_id = fields.Many2one(comodel_name="ir.model", required=True)
+    model_name = fields.Char(
+        related="model_id.model", store=True, readonly=True, index=True,
+        string="Model name",
+    )
+    name = fields.Char(required=True, translate=True)
+    expression = fields.Char(required=True)
+    position_after = fields.Char(
+        help="Optional field name for putting the filter after that one. "
```

Comment:
```suggestion
        help="Optional field name for putting the filter after another. "
```

Reply:
Please read it again. The sentence is saying exactly the purpose.